src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -293,4 +303,41 @@ private static WebACAuthorization createAuthorizationFromMap(final Map<String, L
             return Optional.empty();
         }
     }
+
+    public List<WebACAuthorization> getDefaultAuthorizations() {
+        final Map<String, List<String>> aclTriples = new HashMap<>();
+        final List<WebACAuthorization> authorizations = new ArrayList<>();
+
+        getDefaultAcl().listStatements().forEachRemaining(x -> {
+            if (isAclPredicate.test(x.getPredicate())) {
+                final Triple t = x.asTriple();
+                aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
+                if (t.getObject().isURI()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(t.getObject().getURI());
+                } else if (t.getObject().isLiteral()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(
+                        t.getObject().getLiteralValue().toString());
+                }
+            }
+        });
+
+        authorizations.add(createAuthorizationFromMap(aclTriples));
+
+        return authorizations;
+    }
+
+    private static Model getDefaultAcl() {
+        final String rootAcl = System.getProperty(ROOT_ACL_PROPERTY);
+
+        if (rootAcl != null && new File(rootAcl).isFile()) {
+            LOGGER.debug("Getting Root ACL from file: {}", rootAcl);
+            return RDFDataMgr.loadModel(rootAcl);
+        } else {
+            LOGGER.debug("Getting Root ACL from classpath: {}", ROOT_ACL_LOCATION);
+            final Model model = createDefaultModel();
+            final InputStream is = WebACRolesProvider.class.getResourceAsStream(ROOT_ACL_LOCATION);
+            RDFDataMgr.read(model, is, Lang.TTL);
+            return model;
```

Comment:
Yeah, I suppose you have to get into [this](https://jena.apache.org/documentation/io/rdf-input.html#streammanager-and-locationmapper) to make that work. But at least `try-with-resource`, eh? Actually, I think there are other places in the HTTP API we could probably go back and use `RdfDataMgr::load*` instead of reading into an empty `Model`. Oh, well, some other time.